scipy/optimize/_optimize.py

```diff
@@ -3298,7 +3298,7 @@ def _endprint(x, flag, fval, maxfun, xtol, disp):
 
 
 def brute(func, ranges, args=(), Ns=20, full_output=0, finish=fmin,
-          disp=False, workers=1):
+          disp=False, workers=1, *, integrality=None):
```

Comment:
Integrality sounds like an integral to me. Is there a better name, would `integer` be preferred?

Reply:
We added `integrality` to `linprog` recently because that's what HiGHS calls it. We can still change it, and I'm not really attached to anything, but I'd suggest it be standardized across `optimize`.

Reply:
+1 on `integrality` -- I've most often heard it in my corner of the optimization space as "integrality" constraints

Reply:
`integrality` it is then.